corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -453,3 +453,35 @@ FormplayerFrontend.on('navigateHome', function() {
         FormplayerFrontend.navigate("/apps", { trigger: true });
     }
 });
+
+/**
+ * This is a hack to ensure that routing works properly on FireFox. Normally,
+ * location.href is supposed to return a url decoded string. However, FireFox's
+ * location.href returns a url encoded string. For example:
+ *
+ * Chrome:
+ * > location.hash
```

Comment:
it was just a smaller output so it was an easier example. ill update it to `href` and then truncate the output